content/account/users.textile

```diff
@@ -72,6 +72,32 @@ To remove a user from your account:
 3. Click the *Delete* button next to the user to remove from the account.
 4. Confirm the action.
 
-h2(#delete). Delete your account
-
-Follow "this process":https://faqs.ably.com/how-do-i-cancel-my-account to delete your Ably account. Only an account owner can initiate this process.
+h2(#close). Close your account
+
+You can close your account at any time by downgrading to the "Free package":/docs/pricing/free. The following steps cancel your account:
```

Comment:
[notes] Not sure if we should be more explicit that, this is closing the user's Ably account, such that they won't be able to login after this process. There *might* be some confusion that this is just closing a specific account within their Ably user account.

Also, it might be worth noting that, the user will also be removed from any associated accounts, i.e if they're an account user an another account, they will be removed.

Reply:
done 👉 ff291a8acb1c489c95e5b4bc1a1ab7047b03e973